pkg/common/ac_operations.go

```diff
@@ -278,15 +279,21 @@ func (a *ACOperationsImpl) tryToFindAC(acs []*accrd.AvailableCapacity, storageCl
 	var (
 		allocatedCapacity int64 = math.MaxInt64
 		foundAC           *accrd.AvailableCapacity
+		driveUUID         = make([]string, 0)
 	)
 	for _, ac := range acs {
+		// Available capacity with system disk location won't be allocated
+		if len(driveUUID) == 0 {
+			driveUUID = a.k8sClient.GetSystemDriveUUID(context.Background(), ac.Spec.NodeId)
+		}
+		driveUUID = append(driveUUID, base.SystemDriveAsLocation)
 		if storageClass != "" {
-			if ac.Spec.Size < allocatedCapacity && ac.Spec.Size >= requiredBytes && ac.Spec.StorageClass == storageClass {
+			if ac.Spec.Size < allocatedCapacity && ac.Spec.Size >= requiredBytes && ac.Spec.StorageClass == storageClass && !util.IsDriveSystem(ac.Spec.Location, driveUUID) {
```

Comment:
Do we need to always ignore system drives? If so, maybe we can do IsDriveSystem check earlier?
 ```
if util.IsDriveSystem(ac.Spec.Location, driveUUID) {
   continue
}
```

Reply:
Same